makefile

```diff
@@ -159,15 +159,22 @@ docker_SRC = github.com/docker/docker
 FORCE:
 
 serviced: $(GODEP)
-serviced: FORCE 
+serviced: FORCE
 	$(GO) build $(GOBUILD_FLAGS) ${LDFLAGS}
 	make govet
 	if [ -n "$(GOBIN)" ]; then cp serviced $(GOBIN)/serviced; fi
 
+buildDockerImage: docker_ok
+	cp web/ui/package.json build
+	cp web/ui/npm-shrinkwrap.json build
+	docker build -t zenoss/serviced-build:$(VERSION) build
+
+pushDockerImage: docker_ok
+	docker push zenoss/serviced-build:$(VERSION)
```

Comment:
No. Both of the targets for `buildDockerImage` and `pushDockerImage` are intended for manual execution (e.g. `make buildDockerImage pushDockerImage`). They only need to be executed when someone needs to refresh the contents of the `zenoss/serviced-build` image.   The regular builds for developers, and the different Jenkins builds do not use these targets and they no longer include a `docker build -t zenoss/serviced-build`